vm/src/hint_processor/builtin_hint_processor/bootloader/types.rs

```diff
@@ -22,7 +28,24 @@ impl PackedOutput {
 }
 
 #[derive(Deserialize, Debug, Clone, PartialEq)]
-pub struct Task {}
+pub struct Task {
```

Comment:
I have a different take on `Task` here: https://github.com/Moonsong-Labs/cairo-vm/pull/22/files#diff-f3ede267dc6ba12446e91386bbf0d625a48eb269a6b8114bb4484c9f2850b448L28-L29

Maybe my `RunProgramTask` can take your `Program` as its data, e.g.

```rust
enum Task {
    RunProgramTask(Program),
    CairoPieTask(_),
}
```

Reply:
Yeah I wanted to implement it like that as well, I'll make changes after this PR to use your model + integrate `CairoPie` in the `CairoPieTask` enum.